src/apps/intel/loadgen.lua

```diff
@@ -14,9 +14,10 @@ local can_transmit, transmit
 
 LoadGen = {}
 
-function LoadGen:new (pciaddress)
-   local o = { pciaddress = pciaddress,
-               dev = intel10g.new_sf({pciaddr=pciaddress}) }
+function LoadGen:new (conf)
+   local o = { pciaddr= conf.pciaddr,
+               dev = intel10g.new_sf({pciaddr=conf.pciaddr}),
+               report_rx = conf.report_rx, }
```

Comment:
I modified LoadGen so it reports on received packets as well as transmitted packets.  As traffic gets bounced to its originator and the NIC is locked by the program running the packetblaster, this was the only possible way I thought of of checking traffic gets actually bounced back.

I missed updating LoadGen docs, if the change finally makes it I will update the docs.